tools/lint/check_git_toplevel_env_scrubbed.py

```
@@ -275,39 +398,69 @@ def _scrubs(node: ast.Call, accepted: Set[str]) -> bool:
     return False
 
 
-def scan(path: Path, repo_root: Path) -> List[Tuple[int, str]]:
-    """Return ``(line, message)`` for every unscrubbed ``--show-toplevel`` call."""
+#: Substrings that make a file worth parsing. The first two are the original
+#: token gates; the rest are the spellings of ``git`` as argv[0], which is what
+#: the widened gate keys on.
+_CHEAP_GATE = tuple(GATED_TOKENS) + ('"git"', "'git'", '"git ', "'git ")
```

Ruff (0.16.3)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)